ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -480,6 +481,36 @@ Rectangle {
         return found[1]
     }
 
+    function getTextWithPublicKeys() {
+        let result = messageInputField.getText(0, messageInputField.length)
+
+        if (mentionsPos.length > 0) {
+            let lastIndex = messageInputField.length + 1
```

Comment:
@faust4exe isn't this variable name misleading a bit if we're looking at what it represents?
From my perspective `lastIndex` should be the last possible value, so `let lastIndex = messageInputField.length - 1`

Reply:
In `mentiosPos` we have `leftIndex` and `rightIndex`, so `lastIndex` for me sounds ok. And the purpose of it is to limit the search of mention  by some value. Also I am searching for `rightIndex < lastIndex` , so `length - 1` will not work with a message ending with a mention. Anyway I added one just to be sure it is bigger than any possible value of indexes in mentions.